sdk/appconfiguration/app-configuration/src/policies/throttlingRetryPolicy.ts

```diff
@@ -12,7 +12,7 @@ import {
   Constants,
   RestError
 } from "@azure/core-http";
```

Comment:
Now we just need to move to corev2 so we can fix https://github.com/Azure/azure-sdk-for-js/issues/6484